scripts/cmd.sh

```diff
@@ -16,12 +16,15 @@ chainlinkd query chainlink getLatestFeedData "testfeedid1" --chain-id testchain
 # Query the latest round of feed data
 chainlinkd query chainlink getLatestFeedData --chain-id testchain -o json
 
-# List all module owner
-chainlinkd query chainlink getModuleOwnerList --chain-id testchain -o json
-
 # List existing keys
 chainlinkd keys list --keyring-backend test
 
+# List all module owner
+chainlinkd query chainlink getModuleOwnerList --chain-id testchain -o json
+
 # Add new module owner
-chainlinkd tx chainlink addModuleOwner "cosmos12uxqzq7aae5ew2rksy0cl0cua7e75cu5t3rjxf" "cosmospub1addwnpepqw2xec9wutvxdhvgke029v3hx97jdpzqpr6d33jx62drd92qjxxrsm9t7q5" --from cosmos1mxea03k4y8n7c4fyeseu3c889ejw7kncwhp0uv --keyring-backend test --chain-id testchain
+chainlinkd tx chainlink addModuleOwner "cosmos1wjthz4kmkcusava94f55pg06cqrlxm889udgjn" "cosmospub1addwnpepq00vr93dx4k88rpfupm5wxv50nmq69chxlfal279cexxjy8yl29dc2kqqqn" --from alice --keyring-backend test --chain-id testchain
+
+# module ownership transfer
```

Comment:
it is a bit unclear what this test is for. can you add as a comment the transfer that should occur?
ex: `# module ownership transfer from Alice to Bob`

Reply:
the cosmos addr and pubkey are too arbitrary to understand here

Reply:
ok

Reply:
the address and key are always different when adding a new key